install/install.qmd

```diff
@@ -283,27 +284,71 @@ Common Points of failure; how to address them.
 
 See CmdStan Guide section [Troubleshooting the Installation](https://mc-stan.org/docs/cmdstan-guide/installation.html#troubleshooting-the-installation){target="_blank"} for further details.
 
-## Introductory Notebooks, Vignettes, and Tutorials
-
-### Python
-
-* CmdStanPy User's Guide Examples
+## High-level Stan Interfaces
 
-* "Getting Started with Bayesian Statistics using Stan and Python" by Bob Carpenter
+| **Language** | **Tool** | **Description** |
+|--------------|----------|-----------------|
+| **R**        | [**BRMS**](https://paul-buerkner.github.io/brms/){target="_blank"} | Use extended `lme4`-like formula syntax to specify and fit multivariate and multilevel models in Stan. *(Requires CmdStanR and C++ compiler.)* |
+| **R**        | [**RStanArm**](https://mc-stan.org/rstanarm){target="_blank"} | Provides stable, efficient Stan versions of R model-fitting packages. *(Stan models are pre-compiled, no C++ compiler needed.)* |
+| **R**        | [**Rethinking**](https://github.com/rmcelreath/rethinking){target="_blank"} | Accompanies the book and course materials for [Statistical Rethinking, 2nd Ed](https://www.taylorfrancis.com/books/mono/10.1201/9780429029608/statistical-rethinking-richard-mcelreath){target="_blank"} by Richard McElreath.|
+| **Python**   | [**Bambi**](https://bambinos.github.io/bambi/){target="_blank"} | Bayesian models in Python using formula syntax, built on top of Stan and PyMC. |
+: {tbl-colwidths="[10, 15, 75]"}
 
-* [Multilevel regression modeling with CmdStanPy and plotnine](case-studies/radon_cmdstanpy_plotnine.html){target="_blank"}
 
-### R
-
-* CmdStanR Vignettes
-
-* 2019 Ben Goodrich presentation on hierarchical models with RStan, BRMS, and RStanArm
-
-### General Overview
+## Introductory Notebooks, Vignettes, and Tutorials
 
-* StanCon 2023 "Fundamentals of Stan"
 
-* Course notes: *Software for MCMC* from Odd Kolbjørnsen, Spring 2023, Oslo Uni\
-\fontsize{9pt}{9.4}\selectfont
-www.uio.no/studier/emner/matnat/math/STK4051/v23/timeplan/lecture_12_softwareformcmc.pdf
-\normalsize
+<table>
```

Comment:
Is there not a way to write this using the normal markdown table syntax? I think the HTML one is hard to read, and appreciated how the rest of these changes removed some

Reply:
yes, changed to markdown.